app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/Dialogs/DownloadDialog.tsx

```diff
@@ -109,6 +109,17 @@ export const DownloadDialog = ({
 
     const filePrefix = slugify(`${formName}-${getDate()}`) + "-";
 
+    const extraTranslations = {
+      streetAddress: {
+        en: t("addressComponents.streetAddress"),
+        fr: t("addressComponents.streetAddress"),
+      },
+      city: { en: t("addressComponents.city"), fr: t("addressComponents.city") },
+      province: { en: t("addressComponents.province"), fr: t("addressComponents.province") },
+      postalCode: { en: t("addressComponents.postalCode"), fr: t("addressComponents.postalCode") },
+      country: { en: t("addressComponents.country"), fr: t("addressComponents.country") },
+    };
+
```

Comment:
This stuff shouldn't have to be scattered across unrelated components. Why can't you just i18n?